docs/changelog.md

```diff
@@ -6,6 +6,28 @@
 
 # Changelog
 
+## 0.26.0 (2024-07-08)
+
+### New features
+
+- Handling of receivers internally has been slightly optimized to avoid allocations. ([#1126](https://github.com/pika-org/pika/pull/1126), [#1139](https://github.com/pika-org/pika/pull/1139), [#1192](https://github.com/pika-org/pika/pull/1192))
```

Comment:
Thanks for looking! I like the first better (and better than my version). I've updated it on the release branch since this has already been merged: https://github.com/pika-org/pika/commit/838e3ddabe75540204a7951dc3d6e2ef14312d03.